boards/arm/nrf5340_dk_nrf5340/nrf5340_dk_nrf5340_shared_sram_planning_conf.dts

```diff
@@ -12,6 +12,13 @@
  */
 
 / {
+    chosen {
+      /*
+       * shared memory reserved for the inter-processor communication
+       */
```

Comment:
DTS supports `//` comments as well by the way.

Reply:
good to know, but I think that Zephyr sticks to /* */ commenting style (even for one-liners)